frontends/ol-search-ui/src/factories.ts

```diff
@@ -39,7 +39,7 @@ export const makeSearchResult = (type: string | null) => {
 }
 
 export const makeCourseResult: Factory<LearningResourceResult> = overrides => ({
-  id:                casual.integer(1, 1000),
+  id:                casual.integer(1, 1000).toString(),
```

Comment:
A few times recently we've had CI fail because of id collisions... which seems unlikely given (1,1000) range.

But... I've been user `faker.unique(faker.datatypes.number)` for this sort of thing lately. (I think faker's default range is 1 to 10k)